src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double DEFAULT_RADIUS = 10.0;
+    private static final double MAX_RADIUS = 100;
+    private static final int FIGURE_COUNT = 4;
+    private final Random random = new Random();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureNumber = random.nextInt(FIGURE_COUNT);
```

Comment:
I'd recommend to remove FIGURE_COUNT constant and use FigureNames.values().length here